tssc/step_implementers/create_container_image/buildah.py

```diff
@@ -13,6 +13,9 @@
 | `context`         | True      | `'.'`          | Context to build the container image in
 | `tlsverify`       | True      | `'true'`       | Whether to verify TLS when pulling parent images
 | `format`          | True      | `'oci'`        | format of the built image's manifest and metadata
+| `containers-config-auth-file` | True | `'~/.buildah-auth.json'` | \
```

Comment:
Formatting appears to have been thrown off here

Reply:
it generates correctly. i just didn't want to re-build all the table for the longer variable and then make all the descriptions have like 10 chars to fit in.